orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationResponse.java

```diff
@@ -0,0 +1,49 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.ChatMessage;
+import com.sap.ai.sdk.orchestration.client.model.CompletionPostResponse;
+import com.sap.ai.sdk.orchestration.client.model.TokenUsage;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.annotation.Nonnull;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+
+public record OrchestrationResponse(
+    @Nonnull ChatMessage assistantMessage,
+    @Nonnull List<ChatMessage> allMessages,
```

Comment:
`allMessages` is a notable convenience for end users, this is not natively included in the orchestration response, but needed for chat scenarios or in general for an easy display of prompt + response.